pyne/r2s.py

```diff
@@ -75,7 +75,8 @@ def irradiation_setup(flux_mesh, cell_mats, cell_fracs, alara_params,
                       fluxin="alara_fluxin", reverse=False,
                       alara_inp="alara_inp", alara_matlib="alara_matlib",
                       output_mesh="r2s_step1.h5m", output_material=False,
-                      decay_times=None, sub_voxel=False):
+                      decay_times=None, sub_voxel=False, responses=None,
+                      wdr_file=None):
```

Comment:
should this have a default value? Is the wdr file typically called something specific, or placed somewhere during pyne install? Should it be?

Reply:
The `wdr_file` here should not have a default value. This is a user provided file. This file is not part of pyne, it is for ALARA. Users have to define this file explicitly.